src/lib/planningApplication/progress.tsx

```diff
@@ -40,6 +40,12 @@ import {
   slugify,
 } from "@/util";
 import { contentImportantDates } from "./date";
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
+import isSameOrAfter from "dayjs/plugin/isSameOrAfter";
+dayjs.extend(utc);
+dayjs.extend(isSameOrAfter);
+dayjs.utc().isUTC();
```

Comment:
Did you mean to leave `dayjs.utc().isUTC();` in? 

It's just returning a boolean to say whether its set to UTC or not. ([docs](https://day.js.org/docs/en/plugin/utc#isutc-mode-isutc))